pkg/networkutils/network.go

```diff
@@ -286,24 +313,26 @@ func (n *linuxNetwork) SetupHostNetwork(vpcCIDR *net.IPNet, vpcCIDRs []*string,
 			"-m", "comment", "--comment", "AWS SNAT CHAN", "-j", "AWS-SNAT-CHAIN-0",
 		}})
 
-	for i, cidr := range vpcCIDRs {
+	for i, cidr:= range allCIDRs {
```

Comment:
@mogren I could add two targets to the Makefile: 
- `format` to apply formatting to all go files
- `check-format` to check all go files and list those which require formatting applied.

I could then add the `check-format` target to the travis build, to guarantee that any given commit only goes green if it is properly formatted, as it would fail the build if formatting changes are required. 

If you like this idea, let me know if I should add it as part of this PR, or maybe a different one.

Reply:
(the above would not be applied to the vendor package)

Reply:
If applied now, it would format ~ 11 files, most of them tests, and most of the changes are aliasing `mock` imports to the actual package name. e.g
 `mockec2metadatawrapper "github.com/aws/amazon-vpc-cni-k8s/pkg/ec2metadatawrapper/mocks"`